wzdx/tools/wzdx_translator.py

```diff
@@ -129,7 +129,37 @@ def add_ids(message):
     return message
 
 
-def initialize_wzdx_object(info):
+# Add ids to message
+# This function may fail if some optional fields are not present (lanes, types_of_work, relationship, ...)
+def add_ids(message, event_type):
```

Comment:
Is this specific to v4.0? Should we name it `add_ids_v4` so we know which version we're working with in the future?